core/domain/services/pue/StatsGatherer.php

```diff
@@ -304,4 +308,22 @@ private function paymentMethodStats()
         }
         return $payment_method_stats;
     }
+
+
+    /**
+     * Return a list of active EE add-ons and their versions.
+     *
+     * @return string
+     */
+    private function getActiveAddons()
+    {
+        $activeAddons = [];
+        $addOns = EE_Registry::instance()->addons;
+        if (! empty($addOns)) {
+            foreach ($addOns as $addon) {
+                $activeAddons[] = $addon->name() . '@' . $addon->version();
+            }
+        }
+        return implode(',', $activeAddons);
```

Comment:
```suggestion
        return implode(', ', $activeAddons);
```space after comma :smile:

Reply:
oh, I think I saw in the ee.com code that this isn't what's presented to humans anyway, so space after the comma is actually probably a bad idea as it will interfere with exploding on commas